### PR TITLE
New features and ui to listvotings

### PR DESCRIPTION
**Pull request by zyngielg**
- enhanced layout of the listvotings
- enhanced css of listvotings
- added feature: pagination
- added feature: filtering by voting title
- added feature: filtering by bollean: display inaccessible votes 

I already detected that after X typings, from time to time filter acts slower  - I will add option to search after enter button or sth.

1. Take a look at the code if there are no disasters.
2. Would you want something different in LAYOUT?
3. Would you like something different in COLOURS/FONTS?
4. Personally, I do not like the Category button. I think I am more a fan of the one in CreteVoteForm, do you agree?
5. Should the toggler have the icons? (imo no). Should the toggler have light green colour/light blue colour or this green is ok? (imo both light colours match: blue -> loading screens, green -> already voted, but I think light blue ftw).

Please respond in the comment with the points :)

SS:
image
image
image
image

### CODE REVIEW

File: client/src/components/listvotings/CategoryDropdown.tsx [hunk 1]
```diff
@@ -57,24 +58,32 @@ export default class CategoryDropdown extends Component<ICategoryListProps, ICat
...
-                </MenuItem>
-              );
-            })}
-          </DropdownButton>
-        ) : (
-          <h2>Fetching data from blockchain...</h2>
-        )}
+        <FormGroup>
+          <ControlLabel style={{ fontSize: "1.5em" }}>Category</ControlLabel>
+          <HelpBlock>Select an existing category from the list.</HelpBlock>
```
Comment (akwodkiewicz) [inline client/src/components/listvotings/CategoryDropdown.tsx]: Remove dot

File: client/src/components/listvotings/ListVotingsPanel.tsx [hunk 2]
```diff
@@ -95,27 +101,44 @@ export default class ListVotingsPanel extends Component<IListVotingsPanelProps,
...
               setchosenPrivacySettingInParent={this.setPrivacySetting}
             />
-          </ButtonToolbar>
+          </Col>
         </Row>
-        <Row>
+        {this.state.chosenCategoryIndex != null && (
+          <Row>
+            <Col md={6}>
+              <HelpBlock>Filter the results.</HelpBlock>
```
Comment (akwodkiewicz) [inline client/src/components/listvotings/ListVotingsPanel.tsx]: Remove the ⚫️

File: client/src/components/listvotings/PrivacyButtons.tsx [hunk 3]
```diff
@@ -23,31 +23,36 @@ export default class PrivacyButtons extends Component<IPrivacyButtonsProps> {
...
-        name="privacySetting"
-        value={this.props.chosenPrivacySetting}
-        onChange={this.handleOnChange}
-      >
-        <ToggleButton
-          value={PrivacySetting.All}
-          {...(this.props.chosenPrivacySetting === PrivacySetting.All ? { active: true } : null)}
+      <FormGroup>
+        <ControlLabel style={{ fontSize: "1.5em" }}>Voting type</ControlLabel>
+        <HelpBlock>Display votings with selected privacy type.</HelpBlock>
```
Comment (akwodkiewicz) [inline client/src/components/listvotings/PrivacyButtons.tsx]: No ⚫️ please

**Comment by zyngielg** [comment]
I will adress it in the evening (especially visual parts, when we will be able to contact and exchange the screenshots quickly)

**Comment by zyngielg** [comment]
For now let's leave category dropdown as it is. I wanted to use https://react-select.com/home#custom-styles for both category picking in CreateVote and ListVotings, but I cannot change the fade of the border in the suggested component (borders in react-bootstrap are with the css:
`.form-control:focus {
    border-color: #66afe9;
    outline: 0;
    -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102,175,233,.6);
    box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102,175,233,.6);
}
`
and I cannot set the box-shadow via component :/

Toggle switch changed to light-blue.

**Pull request merged**